### ISSUE TITLE
Adding reaction emoji closes spoiler tag

### ISSUE THREAD

**Issue by WesleyAC** [zulip/zulip-mobile#4527]
If you open a spoiler tag in a message, and then add a reaction emoji, the spoiler tag closes, causing the message window to jump. Instead, the spoiler should remain open.

**Comment by chrisbobbe** [comment]
Yeah, that'd be good to fix. It's a consequence of #3851; I owe more work in the direction of #3913 for that. I've recently picked some of it back up, in #4406, but this is a good reminder that I should follow through. 🙂
